examples/data/interpolation_evalscript.js

```diff
@@ -0,0 +1,176 @@
+//VERSION=3
+
+// Calculate number of bands needed for all intervals
+// Initialize dates and interval
+// Beware: in JS months are 0 indexed
+var start_date = new Date(2020, 6, 1, 0, 0, 0);
```

Comment:
Yes, I agree. It would be even better if one could access a time interval, requested in processing/batch request, from within the evalscript.

Reply:
Should this be done here, or can it wait for future? As your suggestion is out of scope (and literally out of our hands), we can only do the template thing... But we should talk to SH guys.

Reply:
What I like in the current version of `interpolation_evalscript.js` is that it is a valid piece of Javascript code. I definitely wouldn't want to use `string.format` method to insert dates because then all single curly brackets (`{`, `}`) would have to be replaced with double curly brackets and the script would look horrible. Maybe it could be formatted with `datetime.datetime.strftime` or even with `string.replace`? But in either case, the `interpolation_evalscript.js` would not be a completely valid JS code anymore.

Reply:
Very good point. 

I suggest adding a box to notebook with the warning that in case of different time window, the evalscript should be changed accordingly. 

Also, SH has just been updated to allow access of tile geometry and tile envelope can from evalscript.  Perhaps they do plan on doing something similar for time window...

Reply:
I updated the warning comment to be a bit more explicit. But I did leave it in the code cell as it is tightly connected to those few lines of code.

Reply:
I'd still prefer a warning box, like e.g. in `ogc_request.ipynb` example. I might be biased, but when running notebook, I tend to just skim through text, and I often miss such important notices, particularly if they are *commented out code*.

I suggest: 
```md
<div class="alert alert-warning">

**Warning:**
    
Parameters in `evalscript` and `time_interval` are both defined for the same time interval. If you decide to change the time interval you have to change it both here and in evalscript.
</div>
```

Reply:
Ok, done. Before I was sceptical about this because this is only specific to this particular batch use-case and not to batch in general.